src/middlewared/middlewared/plugins/ftp.py

```diff
@@ -0,0 +1,84 @@
+from middlewared.schema import accepts, Bool, Dict, Int, Str
+from middlewared.validators import Exact, IpAddress, Match, Or, Range
+from middlewared.service import SystemServiceService, ValidationErrors, CallError
+
+
+class FTPService(SystemServiceService):
+
+    service_name = "ftp"
+    key_prefix = "ftp_"
```

Comment:
Perhaps we can use the existing Config subclass here:

class Config:
   datastore_prefix = 'ftp_'

What do you think?